SU-SWS/saa_victoria_subtheme
# ADAPT-000: Set Proxima nova for body.

# READY FOR REVIEW

# Summary
- Set body to be proxima nova, with mixin.

# Review By (Date)
- Tues 9/8 if possible please

# Urgency
- medium/high, being noticed right now by clients

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Test that proxima nova is being set and overriding the apple font line, it happens on this page:
https://associates.alumni.stanford.edu/
I have my branch currently on this page to preview ^ But please test locally as well.
Note that there are still areas that need specific font settings, and i'm working on that later today.
![image](https://user-images.githubusercontent.com/12898896/92502810-9f2f1f80-f1bd-11ea-8d3b-4441f08bbd63.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
